Core/Src/gnwmanager.c

```diff
@@ -111,6 +128,9 @@ struct gnwmanager_comm {  // Values are read or written by the debugger
             uint8_t expected_hash[32];
 
             uint8_t actual_hash[32];
+
+            // File path if write to filesystem
+            uint8_t file_path[256];
```

Comment:
I think this is unused, so we should remove it (seems like the prior `work_context_t.file_path` is the one that is actually used).

Reply:
resolved in commit 4693d02aa9303090cead601f9952a7d3ffd2bb4b